Gemfile.lock

```diff
@@ -309,4 +307,4 @@ DEPENDENCIES
   xcpretty
 
 BUNDLED WITH
-   2.4.7
```

Comment:
I also upgraded `bundler` on my machine, but it's still behind whatever version whoever did this last is on. What version of ruby are you running locally @brustolin and @philipphofmann ? I'm on `2.7.5`.

Reply:
2.6.10

Reply:
Huh, they're still shipping newer patches for earlier minor releases. Interesting!
- 2.7.5 released 24 Nov 2021
- 2.6.10 released 12 Apr 2022